### PR TITLE
[UPSTREAM] Fix runtime for gun keybinds

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

from https://github.com/tgstation/TerraGov-Marine-Corps/pull/7001

## Changelog
:cl:
fix: Fix runtime for gun keybinds
/:cl:

### CODE REVIEW

File: code/modules/projectiles/gun_system.dm [hunk 1]
```diff
@@ -215,7 +215,7 @@
 /obj/item/weapon/gun/removed_from_inventory(mob/user)
 	if(!gun_user)
 		return
-	UnregisterSignal(gun_user, list(COMSIG_MOB_MOUSEDOWN, COMSIG_MOB_MOUSEUP, COMSIG_MOB_MOUSEDRAG))
+		UnregisterSignal(gun_user, list(COMSIG_MOB_MOUSEDOWN, COMSIG_MOB_MOUSEUP, COMSIG_MOB_MOUSEDRAG, COMSIG_KB_RAILATTACHMENT, COMSIG_KB_UNDERRAILATTACHMENT, COMSIG_KB_UNLOADGUN))
```
Comment (ZephyrTFA) [inline code/modules/projectiles/gun_system.dm]: Did you mean to indent this further?

Comment (ZephyrTFA) [reply code/modules/projectiles/gun_system.dm]: Because as of right now this never gets run due to being directly after a `return`

Comment (BraveMole) [reply code/modules/projectiles/gun_system.dm]: oh no, not sure what happened

Comment (BraveMole) [inline code/modules/projectiles/gun_system.dm]:
```suggestion
	UnregisterSignal(gun_user, list(COMSIG_MOB_MOUSEDOWN, COMSIG_MOB_MOUSEUP, COMSIG_MOB_MOUSEDRAG, COMSIG_KB_RAILATTACHMENT, COMSIG_KB_UNDERRAILATTACHMENT, COMSIG_KB_UNLOADGUN))
```

**Pull request closed without merging**